README.md

```diff
@@ -14,6 +10,14 @@ Add the following to your `plugins.sbt`:
 addSbtPlugin("com.github.sbt" % "sbt-github-actions" % <latest>)
 ```
 
+**Note**: We recommend you set the following setting in `build.sbt`:
```

Comment:
@eed3si9n Since https://github.com/sbt/sbt-github-actions/pull/136 was merge would it be possible to update this section. Note that I ended up deciding on reverting to the default to JDK 8 with no additional JDK's so you can say something along the lines of

```scala
ThisBuild / githubWorkflowJavaVersions += JavaSpec.temurin("17")
```

Which would actually result in publishing for JDK 8 but building for JDK 8 and JDK 17